app/models/list.rb

```diff
@@ -0,0 +1,7 @@
+class List < ActiveRecord::Base
+  belongs_to :user
+  has_many :items
+
+  validates :title, length: { minimum: 1, maximum: 25 }, presence: true
+  validates :user, presence: true
```

Comment:
Nice, we chatted about these being better as a database validation rather than a Rails validation.